kickstart.sh

```diff
@@ -304,8 +321,6 @@ git="$(which_cmd git)"
 NETDATA_SOURCE_DIR=
 if [ ! -z "${git}" -a -x "${git}" ]
 then
-    SOURCE_DST="/usr/src"
-
     [ ! -d "${SOURCE_DST}" ] && run ${sudo} mkdir -p "${SOURCE_DST}"
 
     if [ ! -d "${SOURCE_DST}/netdata.git" ]
```

Comment:
That makes sense, I''ll move all another 4 netdata hosts to from `/opt/netdata` to `/opt/netdata.git`. Just hate inconsistency.